examples/step-89/step-89.cc

```diff
@@ -1371,7 +1400,7 @@ namespace Step89
 
                 // TODO: Only the information of the quadrature size is needed
                 // (MARCO/PETER)
-                global_quadrature_vector[bface] =
+                global_quadrature_vector[bface] = // PM: this is odd!? Why do wee need empty quadratures?
```

Comment:
This is quite odd, and I would like to discuss options here. The interface is motivated by NonMatching::MappingInfo, where empty quadrature rules are passed if a face is not considered. We can simply use a different interface, I guess.

Reply:
Let's discuss this later when I go through `FERemoteEval` :+1: